config/configCommon.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
 	asyncCompareLimit: 25,
 	engine: 'puppeteer',
 	engineOptions: {
-		headless: 'new',
+		headless: true,
```

Comment:
I think this is the fix for https://phabricator.wikimedia.org/T368476

Reply:
Unsure why

Reply:
Turned out this https://phabricator.wikimedia.org/T368476#9932920 was also in play for the "reference file not found" errors